packages/connectivity/example/lib/main.dart

```diff
@@ -71,10 +71,10 @@ class _MyHomePageState extends State<MyHomePage> {
     // message was in flight, we want to discard the reply rather than calling
     // setState to update our non-existent appearance.
     if (!mounted) {
-      return;
+      return Future.value(null);
```

Comment:
I have always found this weird. It there an open Dart issue about improving this?

Future<void> functions need to be able to actually return void just like Future<int> functions can just `return 5`.

Reply:
It looks like dart-lang/sdk#31278 has it. This looks like it's fixed on master, but our CI runs on both stable and master so for now unfortunately I think I need to keep this version.